src/bitcoin-cli.cpp

```diff
@@ -712,6 +714,9 @@ static UniValue ConnectAndCallRPC(BaseRequestHandler* rh, const std::string& str
     UniValue response(UniValue::VOBJ);
     // Execute and handle connection failures with -rpcwait.
     const bool fWait = gArgs.GetBoolArg("-rpcwait", false);
+    const int timeout = gArgs.GetArg("-rpcwaittimeout", DEFAULT_WAIT_CLIENT_TIMEOUT);
+    const int64_t deadline = GetTime() + timeout;
```

Comment:
I think `GetTime` is deprecated in favor of `GetSystemTimeInSeconds` (not mockable) or `GetTime<T>` (mockable), see src/util.time.h

Reply:
Ok, amended the use of `GetTime` with the `GetTime<T>` version (had to look it up, hopefully I got the right version).